data/reports/m1_proposal.md

```diff
@@ -0,0 +1,41 @@
+
+### Section 2: Data Description
+
+We will be developing a dashboard to visualize the **global prevalence of Tuberculosis (TB) over time** and the **associated risk factors per country**.
+
+We are using two datasets from the World Health Organization (WHO) which can be found [here](https://www.google.com/url?q=https://extranet.who.int/tme/generateCSV.asp?ds%3Destimates&sa=D&source=docs&ust=1711469479701182&usg=AOvVaw2Uui8IYP7fyZ3E0_nXPnGw) and [here](https://www.google.com/url?q=https://extranet.who.int/tme/generateCSV.asp?ds%3Destimates_age_sex&sa=D&source=docs&ust=1711469600536959&usg=AOvVaw3O2Ts9QUTKv5xk08Arn6fE). The first dataset summarizes global trends of TB for 217 countries from 2000 to 2022.
+
+-   Geographic and demographic information: `iso2 code`
+
+-   Annual data per country (identified by `year`):
+
+    -   Population (`e_pop_num`),
+
+    -   WHO-generated estimates:
+
+        -   Absolute estimates of TB incidence with and without HIV: (`e_inc_num`, `e_inc_tbhiv_num`),
+
+        -   Absolute estimates TB-HIV coinfection (`e_inc_tbhiv_num`)
+
+        -   Absolute estimates of mortality in the population with and without HIV: (`e_mort_tbhiv_num`, `e_mort_exc_tbhiv_num`)
+
+        -   Case fatality ratio (`cfr`): calculated as mortality/incidence
+
+We will visualize the different WHO-generated estimates to gain an understanding of the epidemiological landscape of TB. Additionally, since HIV is known to strongly increase the risk of TB (Bell & Noursadeghi, 2017), we want to look at **TB incidence, mortality** (for the population with and without HIV) in conjuction with the **TB-HIV coinfection estimates** to visually display the interplay between the two.
+
+The second dataset summarizes the TB incidence estimates for 215 countries (missing data for 'Netherlands Antilles', 'Serbia & Montenegro') aggregated by **Age, Sex and Risk factors** for the year 2022. Age: '0-14', '0-4', '15-24', '15plus', '18plus', '25-34', '35-44', '45-54', '5-14', '55-64', '65plus' Sex: ‘female’, ‘male’ Risk factors: alc=Harmful use of alcohol; dia=Diabetes; hiv=HIV; smk=Smoking; und=Undernourishment. Finally, we want to visualize other global risk factors associated with TB as measured by the incidence estimates. Unfortunately, since we only have this data for 2022, we will not be able to look at the effects of these risk factors over time.
```

Comment:
"Finally, we want to visualize other global risk factors associated with TB as measured by the incidence estimates"

Which other risk factors here?